_pages/en/about-us.html

```diff
@@ -5,7 +5,23 @@
 lang: en
 permalink: /about-us.html
 ---
+<div class="team-intro">
+<p>This website is a platform where we, IT Strategy, are sharing our plan for becoming a digital government, our progress, and all the tools, information and other resources needed to support this transformation across IITB and ESDC as a whole. 
+It is a place that gives us an opportunity to express our professional opinion and ideas.</p>
 
+<p>The ESDC IT Strategy team is working in the open and welcomes any feedback while work is still in progress.
+Contributing to work in progress is framing our learning experience and differs from formal committee reviews that most government employees are familiar with. 
```

Comment:
"Contributing to work in progress [...]", I guess we mean "Working in the open"?
If so, I would simply say that, as we kind of always "contribute to work in progress" when we work on documents, whether in open or closed environments ;)
Then the next sentence could be "It is a culture..."

Reply:
this comment is not clear to me. Would "Contributing **to work in the open** is framing our learning experience..."  work?

Reply:
I'm not sure if that sounds right (as we switch to "work" as a noun), but I'm not a reference ;)
I was suggesting to instead say "**Working in the open** is framing our learning experience..." if that's the meaning that was intended.